[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-2 (#115637)
openshift/openshift-docs: modules/mirror-registry-introduction.adoc

[id="mirror-registry-introduction_{context}"]
= Mirror registry for Red Hat OpenShift introduction

[role="_abstract"]

> For disconnected deployments of {product-title}, a container registry is required to carry out the installation of the clusters. To run a production-grade registry service on such a cluster, you must create a separate registry deployment to install the first cluster. The _mirror registry for Red{nbsp}Hat OpenShift_ addresses this need and is included in every {product-title} subscription. It is available for download on the OpenShift console Downloads page.

Reviewer: Maybe just a quote around here to hint users to look for the link:

```suggestion
For disconnected deployments of {product-title}, a container registry is required to carry out the installation of the clusters. To run a production-grade registry service on such a cluster, you must create a separate registry deployment to install the first cluster. The _mirror registry for Red{nbsp}Hat OpenShift_ addresses this need and is included in every {product-title} subscription. It is available for download on the "OpenShift console Downloads" page.
```

Because the _mirror registry for Red{nbsp}Hat OpenShift_ uses local storage, you should remain aware of the storage usage consumed when mirroring images and use {quay}'s garbage collection feature to mitigate potential issues. For more information about this feature, see "{quay} garbage collection".
====

> * Support for Red{nbsp}Hat product images that are pushed to the _mirror registry for Red{nbsp}Hat OpenShift_ for bootstrapping purposes are covered by valid subscriptions for each respective product. A list of exceptions to further enable the bootstrap experience can be found on the Self-managed Red{nbsp}Hat OpenShift sizing and subscription guide.

Reviewer: ```suggestion
* Support for Red{nbsp}Hat product images that are pushed to the _mirror registry for Red{nbsp}Hat OpenShift_ for bootstrapping purposes are covered by valid subscriptions for each respective product. For a list of exceptions to further enable the bootstrap experience, see "Self-managed Red{nbsp}Hat OpenShift sizing and subscription guide".
```
